### PR TITLE
Week 2 Step 4

### PR DESCRIPTION
**Pull request by TheArijitSaha**
Added Comments functionality:
- Posting Comments
- Viewing posted comments

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
+          .innerHTML = listHTML;
 }
+
+/*
+ * Functions to be called after
+ * the window has loaded
+ */
+window.onload = () => {
+  getComments();
+}
```
Comment (swetankmodi) [inline portfolio/src/main/webapp/script.js:34]: nit: newline at the end of every file.

**Review by swetankmodi — APPROVED** [review]
LGTM.

**Comment by swetankmodi** [comment]
The commit messages in this PR were very useful and to the point. Maybe you can just list those down in the PR description as well. Thanks!

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (TheArijitSaha) [reply portfolio/src/main/webapp/script.js:34]: Sure, I will add that.

**Pull request merged**